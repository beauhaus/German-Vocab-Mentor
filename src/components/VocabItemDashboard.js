import React from 'react';
import styled from 'styled-components';
import VocabItemList from './VocabItemList';
import VocabListFilters from './VocabListfilters'

const Styled_Template = styled.div`
     width: 100vw;
    height: 100vh;
    grid-column: 1;
    grid-row: 2/-1;
    background: #364958;
`;

const VocabItemDashboard = (props) => (
    <Styled_Template>
    {/* 
    */}
        <VocabListFilters />
        <VocabItemList />
    </Styled_Template>
)


export default VocabItemDashboard;