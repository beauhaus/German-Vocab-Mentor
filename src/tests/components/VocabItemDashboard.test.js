import React from 'react';
import {shallow} from 'enzyme';
import VocabItemDashboard from '../../components/VocabItemDashboard'


test('should render dashboard correctly',() => {
    const wrapper = shallow(<VocabItemDashboard/>)
    expect(wrapper).toMatchSnapshot();
})