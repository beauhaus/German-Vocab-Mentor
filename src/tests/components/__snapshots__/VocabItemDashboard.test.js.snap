// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly 1`] = `
<.sc-bdVaJa>
  <Connect(VocabListFilters) />
  <Connect(VocabItemList) />
</.sc-bdVaJa>
`;
